docker/ci-env/Dockerfile

```diff
@@ -15,7 +15,7 @@
 # specific language governing permissions and limitations
 # under the License.
 
-FROM apache/pegasus:build-env-centos7
+FROM apache/pegasus:build-env-centos7-master
```

Comment:
This Dockerfile seems not used any more, it can be safely removed later.

Reply:
That's true, I've also not found where this image is referenced. I could create another pull requests to remove it.